### PR TITLE
Revert "Upgrade GraalVM to 19.3.0"

### PR DESCRIPTION
**Pull request by gsmet**
This reverts commit 8e20d02f9059fae2325aa218d49f0fb8a16af64b.

Well, and tweaks a few things here and there because there were several conflicts and a few adjustments to be made.

I haven't tested a full native builds as there's no chance I can do that on my laptop. So let's pray and let CI do its magic.

Note: this PR targets 1.1, we still use 19.3 in master.

### CODE REVIEW

**Comment by gsmet** [comment]
@johnaohara can you see if the RSS numbers are back to normal with this PR?

**Comment by johnaohara** [comment]
@gsmet will check RSS numbers

**Comment by johnaohara** [comment]
@gsmet Using this branch, startup RSS for getting started set with -Xmx2m is 13.5MB, compared to 63.5MB with 19.3.  This is consistent with the RSS numbers before the upgrade to 19.3.

**Comment by gsmet** [comment]
OK, cool, thanks. Let's wait for CI then.

File: core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java [hunk 1]
```diff
@@ -325,11 +327,17 @@ public NativeImageBuildItem build(NativeConfig nativeConfig, NativeImageSourceJa
...
         if (vmVersionIsObsolete) {
             throw new IllegalStateException(
-                    "Out of date build of GraalVM detected: " + version + ". Please upgrade to GraalVM 19.3.0.");
+                    "Out of date version of GraalVM detected: " + version + ". Please upgrade to GraalVM 19.2.1.");
+        }
+        final List<String> unsupportedGraalVmVersions = Arrays.asList("19.3.");
+        final boolean vmVersionIsUnsupported = unsupportedGraalVmVersions.stream().anyMatch(v -> version.contains(" " + v));
+        if (vmVersionIsUnsupported) {
+            throw new IllegalStateException(
+                    "Unsupported version of GraalVM detected: " + version + ". Please use GraalVM 19.2.1.");
```
Comment (gwenneg) [inline core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java]: :thumbsup:

**Review by gwenneg — APPROVED** [review]
LGTM

**Pull request merged**